pkg/test/resource_creator.go

```diff
@@ -70,26 +66,26 @@ func (ctx *TestCtx) createFromYAML(yamlFile []byte, skipIfExists bool, cleanupOp
 			return fmt.Errorf("failed to unmarshal object spec: (%v)", err)
 		}
 		obj.SetNamespace(namespace)
-		err = Global.Client.Create(goctx.TODO(), obj, cleanupOptions)
+		err = ctx.client.Create(goctx.TODO(), obj, cleanupOptions)
```

Comment:
The code shows great and is passing in the test which should enough to ensure it.
Just a nit,  it shows be a break changing. Is not it missing the changelog? Also,  should not it goes to a migrate doc? Could we work with an unreleased section as well? WDYT?

Reply:
I don't think this is a breaking change. Did I break something without realizing it?

Reply:
I think we need to change the [doc](https://github.com/operator-framework/operator-sdk/blob/master/doc/test-framework/writing-e2e-tests.md) since it says for the users use the global one and will it `framework.Global` will still be working? If not, then I think is a breaking change.

Reply:
Yep. `framework.Global` is still there and configured the same way. I _think_ there is no change to any public API or functionality at all. Just an internal refactoring to get rid of the unexported global variables.